runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineOptions.java

```diff
@@ -56,202 +58,255 @@ public interface DataflowPipelineOptions
-  @Description("Run the job as a specific service account, instead of the default GCE robot.")
-  String getServiceAccount();
-
-  void setServiceAccount(String value);
-
-  /**
-   * The Google Compute Engine <a
-   * href="https://cloud.google.com/compute/docs/regions-zones/regions-zones">region</a> for
-   * creating Dataflow jobs.
-   */
-  @Description(
-      "The Google Compute Engine region for creating Dataflow jobs. See "
-          + "https://cloud.google.com/compute/docs/regions-zones/regions-zones for a list of valid "
-          + "options.")
-  @Default.InstanceFactory(DefaultGcpRegionFactory.class)
-  String getRegion();
-
-  void setRegion(String region);
-
-  /**
-   * Dataflow endpoint to use.
-   *
-   * <p>Defaults to the current version of the Google Cloud Dataflow API, at the time the current
-   * SDK version was released.
-   *
-   * <p>If the string contains "://", then this is treated as a URL, otherwise {@link
-   * #getApiRootUrl()} is used as the root URL.
-   */
-  @Description(
-      "The URL for the Dataflow API. If the string contains \"://\", this"
-          + " will be treated as the entire URL, otherwise will be treated relative to apiRootUrl.")
-  @Override
-  @Default.String(Dataflow.DEFAULT_SERVICE_PATH)
-  String getDataflowEndpoint();
-
-  @Override
-  void setDataflowEndpoint(String value);
-
-  /** Labels that will be applied to the billing records for this job. */
-  @Description("Labels that will be applied to the billing records for this job.")
-  Map<String, String> getLabels();
-
-  void setLabels(Map<String, String> labels);
-
-  /** The URL of the staged portable pipeline. */
-  @Description("The URL of the staged portable pipeline")
-  String getPipelineUrl();
-
-  void setPipelineUrl(String urlString);
-
-  @Description("The customized dataflow worker jar")
-  String getDataflowWorkerJar();
-
-  void setDataflowWorkerJar(String dataflowWorkerJar);
-
-  /** Set of available Flexible Resource Scheduling goals. */
-  enum FlexResourceSchedulingGoal {
-    /** No goal specified. */
-    UNSPECIFIED,
-
-    /** Optimize for lower execution time. */
-    SPEED_OPTIMIZED,
-
-    /** Optimize for lower cost. */
-    COST_OPTIMIZED,
-  }
-
-  /** This option controls Flexible Resource Scheduling mode. */
-  @Description("Controls the Flexible Resource Scheduling mode.")
-  @Default.Enum("UNSPECIFIED")
-  FlexResourceSchedulingGoal getFlexRSGoal();
-
-  void setFlexRSGoal(FlexResourceSchedulingGoal goal);
-
-  /** Returns a default staging location under {@link GcpOptions#getGcpTempLocation}. */
-  class StagingLocationFactory implements DefaultValueFactory<String> {
-    private static final Logger LOG = LoggerFactory.getLogger(StagingLocationFactory.class);
+    @Description(
+            "Project id. Required when running a Dataflow in the cloud. "
+            + "See https://cloud.google.com/storage/docs/projects for further details.")
+    @Override
```

Comment:
There's a very large update to the Dataflow runner here that is not explained in the PR description.

Is this just formatting ? If so pls revert this so that the doc updates can be reviewed cleanly.

If there are actual updates to the Dataflow runner pls create a separate PR and an Github issue and discuss in Beam dev list appropriately.